### DISCUSSION TITLE
Not downloading audio from some sites

### DISCUSSION THREAD

**Discussion by timekills** [subhra74/xdm#677] [General]
Using XDM under Windows (10 & 11) as well as Ubuntu. Some files from certain sites download the video portion flawlessly but the audio is not downloaded. From the same site, other files download the combined video and audio as expected.
The files from the site worked fine up until recently; doesn't throw an error (as stated, the video portions downloads just no audio track.)
Reproducible regardless of OS being used.

**Comment by subhra74** [comment]
Will be fixed in next release
